purchases/src/main/kotlin/com/revenuecat/purchases/PresentedOfferingContext.kt

```diff
@@ -17,4 +17,14 @@ data class PresentedOfferingContext(
      * The identifier of the placement used to obtain this object.
      */
     val placementIdentifier: String? = null,
+
+    /**
+     * The revision of the targeting used to obtain this object.
+     */
+    val targetingRevision: Int? = null,
+
+    /**
+     * The rule id from the targeting used to obtain this object.
+     */
+    val targetingRuleId: String? = null,
```

Comment:
I was also wondering whether we should have one property of the `Targeting` type, instead of these 2 separately... We could make it internal in order to not expose the type publicly, Wdyt?